README.md

```diff
@@ -41,12 +42,20 @@ $ make
 ```
 This builds the plugin in `/build/rootfs`
 
-## Documentation
 
+### Configuration and Usage
+* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started).
+* Create Global Config, see description in [snap's Global Config] (https://github.com/intelsdi-x/snap-plugin-collector-cinder/blob/master/README.md#snaps-global-config).
+* Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-cinder/blob/master/README.md#examples).
+
+#### Suggestions
+* It is not recommended to set interval for task less than 20 seconds. This may lead to overloading Keystone with authentication requests.
```

Comment:
I think last sentence should be changed to "This may lead to overloading Cinder API with requests"